src/shims/shims.proj

```diff
@@ -94,6 +123,19 @@
       <FileWrites Include="@(BinPlaceItem)" />
     </ItemGroup>
   </Target>
+
+  <Target Name="GetCompatibilityShimsToPackage"
+          BeforeTargets="GetFilesToPackage">
+    <PropertyGroup>
+      <TargetPath>@(ShimsToInclude->'$(GenFacadesOutputPath)%(Identity)')</TargetPath>
```

Comment:
Consider the following:
```xml
<PropertyGroup>
  <TargetPath>@(NETFxReference->'$(GenFacadesOutputPath)%(Identity).dll')</TargetPath>
</PropertyGroup>

<ItemGroup>
  <_manualShimProjects Include="manual\*.csproj" />
  <_manualShims Include="@(_manualShimProjects->'%(FileName)')" />
  <_refShims Include="@(NETFxReference)" Exclude="@(_manualShims)" />
  <FilesToPackage Include="@(_refShims->'$(GenFacadesOutputPath)%(Identity).dll')">
    <TargetFramework>netcoreapp2.0</TargetFramework>
    <TargetPath>ref/netcoreapp2.0/</TargetPath>
  </FilesToPackage>
</ItemGroup>
```

Reply:
I could do the _manualShims calculation, but I can't use NETFxReference as stated in the comment above.